AirCasting/CoreData/UIStorage.swift

```diff
@@ -0,0 +1,69 @@
+// Created by Lunar on 10/02/2022.
+//
+
+import CoreData
+import Combine
+import SwiftUI
+import Resolver
+
+protocol UIStorage {
+    func accessStorage(_ task: @escaping(HiddenCoreDataUIStorage) -> Void)
+}
+
+protocol UIStorageContextUpdate {
+    func cardStateToggle(for sessionUUID: SessionUUID) throws
+    func changeStream(for sessionUUID: SessionUUID, stream: String) throws
+    func save() throws
+}
+
+final class CoreDataUIStorage: UIStorage {
+    
+    @Injected private var persistenceController: PersistenceController
```

Comment:
hmm... Let's maybe not inject it here, instead initialize this class with context and move figuring it out to Resolver's `register` func for `UIStorage`

Reply:
I started to do this, but I am not sure if I get it as it should be. Could you tell me when you will be back from ⛷?
Why do we want to skip Injection in this case?

Reply:
I mean, should I initialize context using the persistenceController that I will require in init?